### PR TITLE
merge image upload input field branch

### PR DESCRIPTION
**Pull request by HenryQuillin**

### CODE REVIEW

**Comment by j-mateo** [comment]
> Visit the preview URL for this PR (updated for commit [ba3c816](https://github.com/projectNEWM/newm-artist-portal/commit/ba3c816a97ebd48dc888778598ed9f1db3557bc9)):
> 
> https://newm-artist-portal--pr13-image-upload-input-f-woehyoyw.web.app
> 
> (expires Sun, 30 Jan 2022 16:25:32 GMT)
> 
> 🔥 via [Firebase Hosting GitHub Action](https://github.com/marketplace/actions/deploy-to-firebase-hosting) 🌎

This is one of those instances when these previews come really handy :D

**Comment by HenryQuillin** [comment]
This is mostly just cleaning up the song upload form component. The image upload component is not yet finished.

File: src/components/ImageUpload.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+interface ImageUploadProps {
+  setFieldValue: (field: string, value: any, shouldValidate?: boolean | undefined) => void;
+  errors: FormikErrors<{
+    description: string;
+    genre: string;
+    imageFile: {
+      name: null;
+      size: null;
+      type: null;
```
Comment (j-mateo) [inline src/components/ImageUpload.tsx:19]: Why are these null types?

File: src/components/ImageUpload.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    releaseDate: string;
+    title: string;
+    yourRole: string;
+  }>;
+}
+
+export const ImageUpload = (props: ImageUploadProps) => {
+  const { setFieldValue, errors, touched } = props;
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({
+    accept: "image/*",
```
Comment (j-mateo) [inline src/components/ImageUpload.tsx:39]: Is there a way to limit the file size using this library?

We'd also want to protect ourselves from decompression bombs. Not in this PR, but something to keep in mind: https://bomb.codes/

**Review by j-mateo — APPROVED** [review]
lgtm! Just a couple of questions

File: src/components/ImageUpload.tsx (hunk 1 above)
Comment (HenryQuillin) [reply src/components/ImageUpload.tsx]: Changed to string types.

**Pull request merged**